tools/cbmc/proofs/run-all-proofs.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/sh
+
+PROOFS_DIR=$(cd "$(dirname "$0")" || exit 1 ; pwd)
+LITANI="${PROOFS_DIR}/../litani/litani"
+
+"${PROOFS_DIR}"/prepare.py
+if [ "$?" -ne 0 ]; then exit 1; fi
+
+"${LITANI}" init --project "Amazon FreeRTOS"
+if [ "$?" -ne 0 ]; then exit 1; fi
+
+N_PROOFS=$(find "${PROOFS_DIR}" -name cbmc-batch.yaml | wc -l)
+CNT=0
+
+echo "Configuring..."
+find "${PROOFS_DIR}" -name cbmc-batch.yaml | while read -r yaml; do
+  PROOF_PATH=$(dirname "${yaml}")
+
+  HARNESS=$(ls "${PROOF_PATH}"/*_harness.c)
+  PROOF_NAME=$(
+    basename "${HARNESS}" |
+    rev                   |
+    cut -c 11-            | # Trim off "_harness.c"
+    rev
+  )
+
+  CNT=$(echo "${CNT} + 1" | bc)
+  printf "\r                                                           "
+  printf "\r%d/%d %s" "${CNT}" "${N_PROOFS}" "${PROOF_NAME}"
```

Comment:
Print to stdout?  Where is stdout being captured?  Is this just progress information?  Add a comment saying so?